games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef

```diff
@@ -0,0 +1,39 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+	major = 0,
+	middle = 0,
+	minor = 0,
+	
+	variety = B_APPV_DEVELOPMENT,
+	internal = 0;
+
+	short_info = "",
+	long_info = ""
```

Comment:
It looks like some rdefs have short descriptions, some others have short copyright notices.
What about:
```
	short_info = "Rocks'n'Diamonds",
	long_info = "Rocks'n'Diamonds, an arcade style game.\n©2001-2013 Artsoft Entertainment"
```